### DISCUSSION TITLE
自动收缩弹视频播放页幕列表(解决了)

### DISCUSSION THREAD

**Discussion by LittleCat01** [the1812/Bilibili-Evolved#2535] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/51882463/142430234-4db59903-a9df-45a4-8bfb-3768744b55be.png)

**Comment by Geeyun-JY3** [comment]
图里想打的是「自动收缩」吧……

这功能 B 站默认就是收起的。
是不是开了脚本的 `自动展开弹幕列表` 功能？

**Reply by LittleCat01** [reply]
是的,
看到了,谢谢提示,是打开了这个功能,关掉就可以了,谢谢
